inst/extdata/tasks_demo/tasks_demo.R

```diff
@@ -51,7 +51,7 @@ demo_task_plan <- function(task_config) {
     task_names=c('AZ','CA','CO'),
     task_steps=list(step1, step2, step3),
     final_steps='report',
-    ind_dir='.',
+    ind_dir='tmp',
```

Comment:
why `tmp` instead of `log` (or no change, for this very simple demo)?

Reply:
I couldn't get "." to work. It couldn't ever find the files because "." went to my home directory. I can make it "log" instead!

Reply:
Interesting. I haven't had a problem with '.' and have been running this a lot this week. We did do some debugging on David's computer, where one time he ran scmake() from within the inst/extdata/tasks_demo folder, which added the .remake folder and other files, which changed the behavior of his tests because everytime `setup_tasks_demo()` was called it was copying the .remake/* files into that new temp directory....doesn't sound like exactly your problem, but I wonder if there's something sorta like that going on.

Reply:
Well, I was doing scmake() from within the `inst/extdata/tasks_demo` folder before the tests were working to check that my new functionality worked. Running scmake from there caused issues with the `.`, but that was all before I ran tests. Changing to "log" fixed everything for me. I don't think it had anything to do with tests because I was having these issues without even touching tests.

Reply:
Maybe it's the nature of how `setwd()` works within the `setup_tasks_demo()` function, so my little workaround to just try out the demo using `setwd("inst/extdata/tasks_demo")` and then `scmake()` didn't work out with "." as planned.

Reply:
Hmmm yeah, just tried running `setup_tasks_demo()` and then my code to test out if it's working with "." as the directory and it worked. I'll revert the folder stuff

Reply:
:+1: